index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles/style.css" type="text/css" rel="stylesheet"/>
+    <!--Link to font-families-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+    <title>Losalini Rokocakau's Portfolio | Remote Full-Stack Developer</title>
+  </head>
+  
+  <body>
+    <!--Navbar for mobile screens-->
+    <header class="nav-mobile">
+      <img src="images/image1.jpg" alt="Geometric image in top corner" class="image-1-mobile"/>
+      <img src="images/ic_menu.svg" alt="Hamburger Icon" class="hamburger-icon"/>
+    </header> 
+
+    <main>
+      <!--Section 1/Headline Section for mobile screens-->
+      <div>
+        <section class="section-1-mobile">
+          <h1 class="section-1-heading-mobile playfair-display-font font-color">Losalini Rokocakau</h1>
+          <p class="section-1-paragraph-mobile poppins-font font-color">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+          If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>  
+        </section>
+        
+        <div class="section-1-button-mobile">
+          <a href="#" class="poppins-font bg-color font-white">Start Collaboration</a>

```

Comment:
- [ ] Please add the `hover` and `pressed ` effect for the `Start Collaboration` button as provided in the Figma design.
![image](https://user-images.githubusercontent.com/24703171/169858531-066e68f7-b132-4376-aa78-eb795c652c2a.png)